### PR TITLE
Version 0.7.0

### PR DESCRIPTION
**Pull request by Laimiux**
Big release

### CODE REVIEW

**Comment by carrotkite** [comment]
### JaCoCo Code Coverage 77.74% :white_check_mark:
| Class | Covered | Meta | Status |
|:---|:---:|:---:|:---:|
| `com/instacart/formula/android/FormulaFragment` | 45% | 0% | :white_check_mark: |


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: build.gradle [hunk 1]
```diff
@@ -26,17 +25,29 @@ buildscript {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
         classpath "com.vanniktech:gradle-android-junit-jacoco-plugin:0.16.0"
         classpath "com.github.ben-manes:gradle-versions-plugin:0.28.0"
-        classpath "org.jetbrains.dokka:dokka-gradle-plugin:0.9.18"
-        classpath "org.jetbrains.dokka:dokka-android-gradle-plugin:0.9.18"
+        classpath "org.jetbrains.dokka:dokka-gradle-plugin:1.4.32"
```
Comment (Laimiux) [inline build.gradle:28]: Fixing issues with dokka.

File: gradle.properties [hunk 2]
```diff
@@ -16,3 +16,5 @@ POM_DEVELOPER_ID=instacart
 POM_DEVELOPER_NAME=Instacart
 
 android.useAndroidX=true
+
+org.gradle.jvmargs=-XX:MaxMetaspaceSize=512m
```
Comment (Laimiux) [inline gradle.properties:20]: Fixing dokka memory issues.

File: formula-android-compose/build.gradle [hunk 3]
```diff
@@ -1,10 +1,7 @@
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
-apply plugin: 'kotlin-kapt'
 apply plugin: "com.github.ben-manes.versions"
```
Comment (Jawnnypoo) [inline formula-android-compose/build.gradle:3]: You shouldn't have to apply this plugin to each module, but rather to the top level build.gradle.

**Review by Jawnnypoo — APPROVED** [review]
Super excited about this!

File: formula-android-compose/build.gradle (hunk 3 above)
Comment (Laimiux) [reply formula-android-compose/build.gradle:3]: Likely did this incorrectly, will update in a follow-up

**Pull request merged**